dockers/docker-fpm-frr/frr/supervisord/supervisord.conf.j2

```diff
@@ -15,6 +15,8 @@ buffer_size=50
 [eventlistener:supervisor-proc-exit-listener]
 command=/usr/bin/supervisor-proc-exit-listener --container-name bgp
 events=PROCESS_STATE_EXITED
+numprocs=5
```

Comment:
not sure you need this, but it seems unnecessary

Reply:
If auto-restart feature was disabled and a critical process was not running, then an event listener will receive this event to generate alerting message continuously (`while loop`). Then a second critical process exited, since we only have one event listener which was in a `while loop`, the `Exited` event of second critical process will not be accepted unless we create one more event listener for it.

So at here for each critical process. there will be an event listener.

Reply:
we should still have only one eventlistener.

meanwhile, change to select mode 

https://repolinux.wordpress.com/2012/10/09/non-blocking-read-from-stdin-in-python/